File: src/Model/WildChild.java
package Model;

import Technical.PowerEnum;

public class WildChild extends GameCharacter {

    public WildChild() {
        super("WildChild",
                "At the start of the game, the Wild Child chooses a mentor. This player is unaware that they have been chosen as the Wild Child’s mentor. If, during the game, the mentor dies, the Wild Child becomes a werewolf.",
                true,
                PowerEnum.Constant,
                "../../resources/images/characters/WildChild.jpg");
    }
}

File: app/src/main/java/Model/WildChild.kt
package Model

class WildChild(order: Int)
    : Character("The Wild Child choose a player that he/she consider as his/her model. If the model dies, the Wild Child become a werewolf. Otherwise, he's a simple villager."
    , true
    , PowerState.PERMANENT
    , false
    , order
    , 1) {

    override fun action(): String {
        isNocturnal = false
        return "The $className choose a player to be his/her master. If the master dies, he/she become a werewolf."
    }

    override fun clone() = WildChild(this.order)
}